app/controllers/password_resets_controller.rb

```diff
@@ -0,0 +1,26 @@
+class PasswordResetsController < ApplicationController
+  def new
+  end
+
+  def create
+    user = User.find_by_email(params[:email])
+    user.send_password_reset if user
+    redirect_to root_path, :notice => "Reset password email sent!"
```

Comment:
Inconsistent. Is there a specific reason the '=>' was used? If not, maybe stick with something like "notice: ..."